src/posts/xrisk.md

```diff
@@ -5,11 +5,9 @@ description: Why AI is a risk for the future of our existence, and why we need t
 
 ## Experts are sounding the alarm
 
-AI researchers on average [believe](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/) there's a 14% chance that once we build a superintelligent AI, it will lead to "very bad outcomes (e.g. human extinction)".
+AI researchers on average [believe](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/) there's a 5% chance that once we build a superintelligent AI, it will lead to "very bad outcomes (e.g. human extinction)".
```

Comment:
The average value is 14%, not 5%.

It's misleading to use the median and say `average`. Why should we mislead people about this?

Reply:
Changed to "The average AI researcher believes".

See discussion above. I think giving conservative estimates is more persuasive, and strategically choosing the highest of the mean or median for each survey is in fact statistical manipulation.

Reply:
Choosing the Lowe number because it's more pursuasive is explicitly manipulative though.


If anything, I'd rather revert to the "half of scientists think there's a larger than 10% chance", which shows that it's not a small number of extremes pushing the number.